meson_options.txt

```diff
@@ -8,3 +8,21 @@ option('run-lint',
        type: 'boolean',
        value: true
 )
+
+option('eosmetrics',
+       description: 'whether to use eosmetrics or not',
+       type: 'boolean',
+       value: true
+)
```

Comment:
I've added some new commits to remove the eosmetrics from the clubhouse and also to improve the data sent to the matomo server.

I've also added a new method to the clubhouse dbus API to use in other apps, like the toolbox, to be able to register metrics on matomo from other components.

There's also another PR related to this one, to use the new API instead of the eosmetrics on the toolbox:     https://github.com/endlessm/hack-toolbox-app/pull/250